File: src/main/java/core/repository/RepositoryInfo.java
package core.repository;

/**
 * Info about {@link RepositoryCommander}. Used for reports generation
 */
public class RepositoryInfo {
    public final String repositoryName;
    public final String username;

    public RepositoryInfo(String repositoryName, String username) {
        this.repositoryName = repositoryName;
        this.username = username;
    }
}

File: src/main/kotlin/com/mirf/core/repository/RepositoryInfo.kt
package com.mirf.core.repository

/**
 * Info about [RepositoryCommander]. Used for reports generation
 */
data class RepositoryInfo(val repositoryName: String, val username: String){

        fun copy(): RepositoryInfo {
            return RepositoryInfo(repositoryName, username)
        }
}
